### PR TITLE
updated README file

### PR DESCRIPTION
**Pull request by thasteris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +59,57 @@ the client within your code instead.
...
+<!-- Authorization: 'api-key': 'your_token' -->
+
+To retrieve transcript-related data using the provided API, you can utilize the Transcript endpoints. These endpoints facilitate the retrieval of transcript data based on a specific transcript name and reference genome. The endpoint structure follows a GET request format and requires two parameters: transcript_name and optionally ref_genome, which defaults to hg19 if not specified. In summary, you can efficiently retrieve transcript-related information, such as coding length, exon details, genomic position, and more, based on the provided transcript name and reference genome, optionally leveraging account benefits through the Authorization header.
+
+To view available request parameters (used by the 'params' method parameter), refer to an example at api.varsome.com.
+
+### Example to query Single Reads
+
+    varsome_api_run.py -g hg19 -k 'api-key' -u https://stable-api.varsome.com -q 'single-read/AGTCCRAGTTGTAAATGGTACACTCGGCGTAAGCCTGAAAAGATAAAATCAAAGATGTAAAGGTGAGCACAGTCTAAGTTCTCTCTGAAGTGTCAATGGGAATGCAGATTGGATTAAATAAATGCTGCCCAAGTGCATACTCAAAGAGGC' -p add-all-data=1
```
Comment (kchatzikok) [inline README.md:97]: Change this to the following:
```
Try the following to retrieve data relevant to a single read:
varsome_api_run.py -g hg19 -k api-key -q 'single-read/AGTCCRAGTTGTAAATGGTACACTCGGCGTAAGCCTGAAAAGATAAAATCAAAGATGTAAAGGTGAGCACAGTCTAAGTTCTCTCTGAAGTGTCAATGGGAATGCAGATTGGATTAAATAAATGCTGCCCAAGTGCATACTCAAAGAGGC' -p add-all-data=1
```

File: README.md [hunk 2]
```diff
@@ -59,12 +59,57 @@ the client within your code instead.
...
+
+To take advantage of your account's benefits you may optionally include your VariantAPI token as a request authorization header. 
+<!-- Authorization: 'api-key': 'your_token' -->
+
+With the single-read, the sequencer reads a fragment from only one end to the other, generating the sequence of base pairs. Single-read sequencing typically involes sequencing a DNA fragment just once, without the need for paired-end sequencing. This approach is favoerd for its cost-effectivness and efficiency in quickly obtain genomic data. By sequencing each DNA fragment only once, single-read sequencing offers a streamlined approach to genome analysis. Also, it is valuable tool in various genomic appliccations, such as indentifying sing nucleotide polymorphirms (SNPs), analyzing gene expression and conducting microbial profiling.
+
+To view available request parameters (used by the 'params' method parameter), refer to an example at api.varsome.com.
```
Comment (kchatzikok) [inline README.md:104]: remove this text

File: README.md [hunk 3]
```diff
@@ -59,12 +59,57 @@ the client within your code instead.
...
+### Example to query Transcripts 
+
+    varsome_api_run.py -g hg19 -k 'api-key' -u https://stable-api.varsome.com -q 'transcript/NM_001276760' -p add-all-data=1
+
+To take advantage of your account's benefits you may optionally include your VariantAPI token as a request authorization header. 
+<!-- Authorization: 'api-key': 'your_token' -->
+
+To retrieve transcript-related data using the provided API, you can utilize the Transcript endpoints. These endpoints facilitate the retrieval of transcript data based on a specific transcript name and reference genome. The endpoint structure follows a GET request format and requires two parameters: transcript_name and optionally ref_genome, which defaults to hg19 if not specified. In summary, you can efficiently retrieve transcript-related information, such as coding length, exon details, genomic position, and more, based on the provided transcript name and reference genome, optionally leveraging account benefits through the Authorization header.
+
+To view available request parameters (used by the 'params' method parameter), refer to an example at api.varsome.com.
```
Comment (kchatzikok) [inline README.md:93]: Please change this to the following
```
Try the following query to retrieve transcript-related data:
varsome_api_run.py -g hg19 -k api-key -q 'transcript/NM_001276760' -p add-all-data=1
```

**Review by kchatzikok — CHANGES_REQUESTED** [review]

**Comment by thasteris** [comment]
@kchatzikok , thank you for your detailed comments. I made the changes.

**Review by bsantor — APPROVED** [review]

**Review by n-zervos — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -59,12 +59,37 @@ the client within your code instead.
 The command above will read variants from `variants.txt` and dump the annotations to `annotations.txt`. 
 For any number of variants you will need to [register](mailto:support@saphetor.com) for an API key. 
 
+### Example to query CNVs
+
+    varsome_api_run.py -g hg19 -k api-key -q https://stable-api.varsome.com -q 'cnv/chr2:83300000:106000000:dup' -p add-all-data=1
```
Comment (kchatzikok) [inline README.md:64]: It should be without the stable-api
```python
varsome_api_run.py -g hg19 -k api-key -q 'cnv/chr2:83300000:106000000:dup' -p add-all-data=1
```

File: README.md [hunk 5]
```diff
@@ -59,12 +59,37 @@ the client within your code instead.
...
+
+The query parameter specifies the CNV's details such as chromosome, start and end positions, and CNV type (deletion or duplication). In this example we have chr2, 83300000 and 106000000 and dup accordingly (for deletion use del).
+
+### Example to query Genes
+
+    varsome_api_run.py -g hg19 -k api-key -q https://stable-api.varsome.com -q 'gene/EGFR' -p add-all-data=1
```
Comment (kchatzikok) [inline README.md:70]:
```python
varsome_api_run.py -g hg19 -k api-key -q 'gene/EGFR' -p add-all-data=1
```

**Review by kchatzikok — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (thasteris) [reply README.md]: Done 👍🏼  @kchatzikok !!!

**Pull request merged**